content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -308,42 +336,47 @@ Open the `influx-handler.json` handler definition provided with the sandbox, and
 
-We can use sensuctl to confirm that the handler has been created successfully.
+Use sensuctl to confirm that the handler was created successfully.
 
 {{< highlight shell >}}
 sensuctl handler list
 {{< /highlight >}}
 
-You should see the `influx-db` handler.
-(If you've completed [lesson \#2](#lesson-2-pipe-keepalive-events-into-slack), you'll also see the `keepalive` handler.)
+The `influx-db` handler should be listed. If you completed [lesson \#2](#lesson-2-pipe-keepalive-events-into-slack), you'll also see the `keepalive` handler.
 
 **4. Create a check to monitor Nginx**
 
-Use the `curl_timings-check.json` file provided with the sandbox to create a service check that runs `metrics-curl.rb` every 10 seconds on all entities with the `entity:sensu-go-sandbox` subscription and sends events to the InfluxDB pipeline:
+The `curl_timings-check.json` file provided with the sandbox will create a service check that runs the `metrics-curl.rb` check plugin every 10 seconds on all entities with the `entity:sensu-go-sandbox` subscription and sends events to the InfluxDB pipeline. The `metrics-curl.rb` plugin is already included as the value of the command field in `curl_timings-check.json` &emdash; you just need to create the file:
 
 {{< highlight shell >}}
 sensuctl create --file curl_timings-check.json
+{{< /highlight >}}
 
+{{< highlight shell >}}
 sensuctl check list
+{{< /highlight >}}
+
+{{< highlight shell >}}
      Name                                        Command                                     Interval   Cron   Timeout   TTL        Subscriptions        Handlers   Assets   Hooks   Publish?   Stdin?     Metric Format      Metric Handlers  
 ────────────── ──────────────────────────────────────────────────────────────────────────── ────────── ────── ───────── ───── ───────────────────────── ────────── ──────── ─────── ────────── ──────── ──────────────────── ───────────────── 
 curl_timings   /opt/sensu-plugins-ruby/embedded/bin/metrics-curl.rb -u "http://localhost"         10                0     0   entity:sensu-go-sandbox                               true       false    graphite_plaintext   influx-db        
 {{< /highlight >}}
 
-This check defines a metrics handler and metric format.
-In Sensu Go metrics are a core element of the data model, so we can build pipelines to handle metrics separately from alerts.
-This allows us to customize our monitoring workflows to get better visibility and reduce alert fatigue.
+This check specifies a metrics handler and metric format. In Sensu Go, metrics are a core element of the data model: you can build pipelines to handle metrics separately from alerts. This allows you to customize your monitoring workflows to get better visibility and reduce alert fatigue.
 
-After about 10 seconds, we can see the event produced by the entity:
+After about 10 seconds, you can see the event produced by the entity:
 
 {{< highlight shell >}}
 sensuctl event info sensu-go-sandbox curl_timings --format json | jq .
```

Comment:
When I run:
`sensuctl event info sensu-go-sandbox curl_timings --format json | jq .`

I get:
`-bash: jq: command not found`

Reply:
When I run:
`sensuctl event info sensu-go-sandbox curl_timings --format json` (without the ` | jq.`), I get the response shown (`"history`" etc.)

Reply:
both `epel-release` and `jq` packages are meant to be installed in https://github.com/sensu/sandbox/blob/master/sensu-go/provision/setup.sh#L79-L80 and https://github.com/sensu/sandbox/blob/master/sensu-go/provision/setup.sh#L99 respectively. I suspect the logic to determine whether `epel-release` is needed is failing us somehow.

Reply:
I'm still having the same problem here.

```
[sensu_go_sandbox]$ sensuctl event info sensu-go-sandbox curl_timings --format json | jq .
-bash: jq: command not found
```

Reply:
Yay it works!